cobalt/worker/service_worker_consts.cc

```diff
@@ -45,5 +45,23 @@ const char ServiceWorkerConsts::kServiceWorkerAllowed[] =
 
 const char ServiceWorkerConsts::kSettingsJson[] =
     "service_worker_settings.json";
+
```

Comment:
This also looks like a change that already is in https://github.com/youtube/cobalt/pull/173/files

Reply:
Yes, it is. I discussed the cherry-pick workflow offline with @oxve, and we thought better to have one PR per one cherry-pick CL. And this happens when I send out the chain PRs, so that I don't need to wait to get the first CL approved then send out this one. The stacked PR included the commits from the base PR, but it should go away when I merge the base PR. Let me try to merge the first PR and verify if it works as expected.

Reply:
Hmm, it does not work like this. Will send the PR one by one so that the stack PR doesn't include all commits.